qiskit/transpiler/passmanager.py

```diff
@@ -197,6 +197,11 @@ def run(  # pylint:disable=arguments-renamed
                     Beware that the keyword arguments here are different to those used by the
                     generic :class:`.BasePassManager`.  This pass manager will translate those
                     arguments into the form described above.
+                .. note::
```

Comment:
Thanks for adding this. Can you please move it to the end of the `callback` parameter section, right after the `def callback_func(**kwargs):` code? Placing it here results in having two consecutive Note comments in the API doc and actually it seems a bit misplaced here anyway.